FreeRTOS-Plus/Source/Application-Protocols/network_transport/freertos_plus_tcp/using_plaintext/using_plaintext.c

```diff
@@ -131,6 +131,14 @@ int32_t Plaintext_FreeRTOS_recv( NetworkContext_t * pNetworkContext,
                                   bytesToRecv,
                                   0 );
 
+    if( socketStatus == -pdFREERTOS_ERRNO_ENOSPC )
```

Comment:
I don' t think it is possible for FreeRTOS_recv() to return -pdFREERTOS_ERRNO_ENOSPC - if there are no bytes available it returns 0 already.

Reply:
Oh right. Good point.